planetiler-core/src/main/java/com/onthegomap/planetiler/util/CommonStringEncoder.java

```diff
@@ -1,41 +1,58 @@
 package com.onthegomap.planetiler.util;
 
+import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.atomic.AtomicInteger;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * A utility for compressing up to 250 commonly-used strings (i.e. layer name, tag attributes) into a single byte.
+ * A utility for compressing commonly-used strings (i.e. layer name, tag attributes).
  */
 @ThreadSafe
 public class CommonStringEncoder {
 
-  private final ConcurrentMap<String, Byte> stringToId = new ConcurrentHashMap<>(255);
+  private final Map<String, Byte> stringToId = new ConcurrentHashMap<>(255);
   private final String[] idToString = new String[255];
   private final AtomicInteger layerId = new AtomicInteger(0);
 
+  private final Map<String, Integer> stringToIdMap = new ConcurrentHashMap<>(255);
+  private final Map<Integer, String> idToStringMap = new ConcurrentHashMap<>();
```

Comment:
Actually if we just raise the limit to 100k, then a `String[]` would only take up 800kb of memory - let's keep the array for idToString instead of concurrent hash map.